### ISSUE TITLE
Add disclaimer to Readme.md that Hardware Accelerated off-screen rendering is not available.

### ISSUE THREAD

**Issue by LiamKarlMitchell** [ryanmcnz/gmweb#7]
This means things such as WebGL and 3D CSS operations may not be very good or supported at all.

**Comment by ryanmcnz** [comment]
Good point. Probably in general it would be good to have a list of what "is" and "isn't" supported, compared to say...a regular WebKit-based browser that is fully updated.
